docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -150,6 +153,379 @@ gene_start      gene_end        symbol  id
 (truncated)
 ```
 
+## Gene Expression Examples
+
+### Gene Expression Availability Endpoint
+
+Retrieve the availability of gene expression data for only case ids, only gene ids, or both case and gene ids. It's also possible to use only case set id, only gene set id or both case set id and gene set id. The availability response informs the user if gene expression data exists for each case or gene.  Gene expression data is only available for protein-coding genes. 
```

Comment:
it seems like the response also tells you how many cases/genes have expression counts and how many don't? if so, it may be worth mentioning this. but not really a big deal.

Reply:
@wwysoc2 would know the language of the docs better. Is it clearer/more consistent, just like the description you give in the table, to refer to this endpoint as being for retrieving data for specific cases, specific genes, or both. And then note that these cases and genes are specified using their case_ids and gene_ids?
In other words, one sentence is about what the endpt does. and the other sentence is about how we specify/refer to the entities in the call/response.

Reply:
changed the description for the Gene Expression Availability Endpoint and a bit for the Values Endpoint, thank you!